### ISSUE TITLE
"Search" not working in online R Book

### ISSUE THREAD

**Issue by jsayers-bsc** [MotusDev/Motus-TO-DO#665]
The "Search" button at the top/left corner of the Motus R Book doesn't seem to be working. All the other links work. I've tried this on a few browsers.

![image](https://user-images.githubusercontent.com/55001144/73226570-9746d180-413e-11ea-8395-06eab0bbe1f8.png)

**Comment by steffilazerte** [comment]
I've update the beta motus R book and tested the search functions locally (which work). 

However, I'm not sure if the book is served automatically from the github site to the <https://beta.motus.org/motusRbook>. 

Could someone confirm that it is served? After that if there's still a problem, I think it may relate to how the book is being served.

**Comment by denislepage** [comment]
we are doing a manual push whenever we are ready to publish updates on either server.

**Comment by Hussell** [comment]
I check for updates on all Motus-related repositories every work-day morning, and deploy things if they haven't already been deployed. I deployed the book to beta this morning, but search still isn't working for me in Firefox. Clicking the button has no effect except appending a # to the URL, there isn't even a JavaScript error. The other buttons in the toolbar work, and the shortcut keys described there also work, except "f" for search.

**Comment by steffilazerte** [comment]
Good to know, I'll look into this a bit more and see if I can find any references of others running into similar problems.

**Comment by steffilazerte** [comment]
I've updated my version of bookdown and recompiled the beta book. A quick test on my personal website (<https://steffilazerte.ca/motusbook/_book>) shows that the search function works. Once it gets posted to the beta.motus.org site we should be able to see if this was a bookdown version problem or a beta.motus.org server problem

**Comment by steffilazerte** [comment]
Whoops, didn't mean to close it

**Comment by Hussell** [comment]
Hmm. I deployed the changed files, but although it's working on your personal website, it still isn't working on the beta server. Maybe not all the necessary files are being transferred? The deployment script currently copies over everything in the _book directory of the repository, including sub-directories. I tried deleting everything and re-uploading, no change.

... I looks like the problem is a 404 on search_index.json, even though the file is present on the server. I'm guessing this is a server configuration issue, i.e. it's ignoring .json files. I'll ask Denis if he can check that.
